### PR TITLE
Add: User page

### PR DESCRIPTION
**Pull request by saurabhchirde**
Desktop View -
Screenshot 2022-05-02 005641

Tablet View -
Screenshot 2022-05-02 005702

Mobile View - 
Screenshot 2022-05-02 005745

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Ui is clean and brilliant


You can go ahead with code

File: src/Components/UI/Footer/Footer.css [hunk 1]
```diff
@@ -8,12 +8,23 @@
 
 .footer-body h3 {
   font-weight: 400;
-  font-size: 0.9rem;
-  color: var(--text-dark);
+  font-size: 0.8rem;
```
Comment (PuneetP16) [inline src/Components/UI/Footer/Footer.css:11]: Suggestion: Could use font size unit as px if the font size is less than 1 rem.

**Review by PuneetP16 — APPROVED** [review]
UI is really good. Nice work. Looks good to merge.

File: src/Components/UI/Footer/Footer.css (hunk 1 above)
Comment (saurabhchirde) [reply src/Components/UI/Footer/Footer.css:11]: I'm using the default root font size of 16px, that's why didn't converted to px.

**Pull request merged**